### PR TITLE
rename (in docs and settings): automation -> campaigns

### PR DESCRIPTION
**Pull request by sqs**
See https://github.com/sourcegraph/about/pull/583 for info on the rename.
    
This only performs the rename in documentation and settings; it does not rename all instances in code (e.g., the `a8n` Go package is not renamed).

### CODE REVIEW

**Comment by ryan-blunden** [comment]
Created https://github.com/sourcegraph/sourcegraph/issues/8512

File: enterprise/internal/a8n/service.go [hunk 1]
```diff
@@ -318,7 +318,7 @@ func RunChangesetJob(
 		CommitInfo: protocol.PatchCommitInfo{
 			Message:     c.Name,
 			AuthorName:  "Sourcegraph Bot",
-			AuthorEmail: "automation@sourcegraph.com",
+			AuthorEmail: "campaign@sourcegraph.com",
```
Comment (unknwon) [inline enterprise/internal/a8n/service.go]: nit
```suggestion
			AuthorEmail: "campaigns@sourcegraph.com",
```

File: doc/user/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ All features:
 - [Usage statistics](usage_statistics.md)
 - [User surveys](user_surveys.md)
 - [Color themes](themes.md)
-- [Automation preview](automation.md)
+- [Code campaigns](campaigns.md)
```
Comment (unknwon) [inline doc/user/index.md]: TBH I feel very inconsistent that we have both "Code campaigns" and "Campaigns" in the documentation and code.  I would prefer we just use "Campaigns" everywhere.

**Comment by mrnugget** [comment]
I think this needs to be rebased after https://github.com/sourcegraph/sourcegraph/pull/8510

File: enterprise/internal/a8n/service.go (hunk 1 above)
Comment (sqs) [reply enterprise/internal/a8n/service.go]: done

File: doc/user/index.md (hunk 2 above)
Comment (sqs) [reply doc/user/index.md]: done

**Pull request merged**